wp-config-pantheon.php

```diff
@@ -96,6 +96,13 @@
  * To override, define your constant in your wp-config.php before wp-config-pantheon.php is required.
  */
 
+if ( ! defined('PANTHEON_HOSTNAME' ) ) {
+    $site_name = $_ENV['PANTHEON_SITE_NAME'];
+    $hostname = ! isset( $_SERVER['HTTP_HOST'] ) ? $_ENV['PANTHEON_ENVIRONMENT'] . "-$site_name.pantheonsite.io" : $_SERVER['HTTP_HOST'];
+    $hostname = $_ENV['PANTHEON_ENVIRONMENT'] === 'lando' ? "$site_name.lndo.site" : $hostname;
```

Comment:
```suggestion
    $hostname = $_ENV['PANTHEON_ENVIRONMENT'] === 'lando' ? "{$site_name}.lndo.site" : $hostname;
```
maybe other languages creeping in here but this feels wrong to me now 😁